modules/rostests/apitests/ntdll/NtWriteFile.c

```diff
@@ -316,4 +400,7 @@ START_TEST(NtWriteFile)
                                  &BufferSize,
                                  MEM_RELEASE);
     ok_hex(Status, STATUS_SUCCESS);
+
+    test_FileAppendData();
```

Comment:
I'd recommend turning the current `START_TEST(NtWriteFile)` into a new function, then adding a new `START_TEST(NtWriteFile)` block that calls both functions.